packages/webpack-cli/lib/utils/warnings/bailAndWatchWarning.js

```diff
@@ -0,0 +1,7 @@
+const logger = require('../logger');
+
+module.exports = (compiler) => {
```

Comment:
Assign this to variable and then assign it to `module.exports`.
Add a comment about function(or file) as well.